lib/mpl_toolkits/mplot3d/axis3d.py

```diff
@@ -47,6 +49,7 @@ def tick_update_position(tick, tickxs, tickys, labelpos):
 
     tick.tick1On, tick.tick2On = True, False
     tick.tick1line.set_linestyle('-')
+    tick.tick1line.set_linewidth(rcParams['grid.linewidth'])
```

Comment:
Is this the correct rc param to use? Shouldn't it be something like `xtick.major.width`? We do have per-axis rcparam handling in axes3d.py, I think.